docs/content/how-tos/rules/FL0062.md

```diff
@@ -24,7 +24,14 @@ Refactor to extract code out into another file.
         "maxLinesInFile": {
             "enabled": false,
             "config": {
-                "maxLinesInFile": 1000
+                "maxLinesInFile": 1000,
+                "skipBlankLines": false,
+                "skipComments": false
             }
         }
     }
+*Introduced in `0.23.0`:*
```

Comment:
@Mersho my god man, can you add an EOL before a heading? this is just common sense